Lib/email/utils.py

```diff
@@ -417,8 +417,13 @@ def decode_params(params):
         for name, continuations in rfc2231_params.items():
             value = []
             extended = False
-            # Sort by number
-            continuations.sort()
+            # Sort by number, treating None as 0 if there is no 0, ignore it if there is already a 0.
```

Comment:
```suggestion
            # Sort by number, treating None as 0 if there is no 0,
            # and ignore it if there is already a 0.
```

Just to fit under 80 chars

Reply:
Nice catch :D